genesyscloud/util/feature_toggles/exporter_sanitizer.go

```diff
@@ -0,0 +1,26 @@
+package feature_toggles
+
+import "os"
+
+const sanitizerLegacy = "GENESYS_SANITIZER_LEGACY"
+const sanitizerOptimized = "GENESYS_SANITIZER_OPTIMIZED"
```

Comment:
Does this mean one or the other has to be set, or does one get defaulted to if no env variables are used?

Reply:
I spoke with @carnellj-genesys about this. We determined that we will remove the `GENESYSCLOUD_SANITIZER_LEGACY` feature toggle with this change. We'll keep the existing implementation as is, and wrap my updates in the new `GENESYSCLOUD_SANITIZER_OPTIMIZED` feature toggle.